internal/pkg/diff/diff.go

```diff
@@ -18,13 +18,26 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/strhelper"
 )
 
+type Option func(c *Config)
+
+type Config struct {
+	allowTargetEnv bool
+}
+
+func WithAllowTargetEnv(allowTargetEnv bool) Option {
+	return func(c *Config) {
+		c.allowTargetEnv = allowTargetEnv
+	}
+}
+
```

Comment:
Here should be `WithIgnoreBranchName`, this part of KAC has nothing directly to do with "allowTargetEnv".

Reply:
[8855283](https://github.com/keboola/keboola-as-code/pull/1835/commits/8855283ad3bcc9f6c1507a6b8e768a6c2472cb82)